src/LiveComponent/src/Storage/SessionStorage.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LiveComponent\Storage;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Implementation of the StorageInterface using Symfony's session mechanism.
+ *
+ * This class provides a session-based storage solution for managing data
+ * persistence in Symfony UX LiveComponent. It leverages the Symfony
+ * `RequestStack` to access the session and perform operations such as
+ * storing, retrieving, and removing data.
+ *
+ * Common use cases include persisting component state, such as form data
+ * or multistep workflow progress, across user interactions.
```

Comment:
>  Common use cases include persisting component state, such as form data, or multistep workflow progress, across user interactions.

I'm having doubt if we should push session usage like this, as in the same time we push for stateless components, removed csrf, etc..

And i'm very hesistant to add such feature in another PR :| 

I don't think these two things should be handled as one... 

Will sleep on it... but already can see several alternatives:
- having an internal/experimental storage (just pour the multi-step form)
- using props
- using existing adapters (doctrine, cache)

Reply:
If it is stateless. okay we could introduce a NullStorage. But in order to be able to reload the page and having a persistent state is quite cool in my opinion.